README.md

```diff
@@ -34,10 +34,10 @@ JSON.stringifyAsync({foo: 1}).then(result => console.log(result));
 Because `JSON.stringifyAsync` and `JSON.parseAsync` are returning promises, they can be used with `await` as a drop-in replacement of sync methods.
 
 ```js
-async ()=> {
+(async ()=> {
   let object = await JSON.parseAsync('{"foo": 1}');
   let string = await JSON.stringifyAsync({foo: 1});
-}();
+})();
```

Comment:
wait, typically that's putting the function in expression position, thus allowing it to be immediately invoked. Are you suggesting that won't work with async functions? That doesn't seem right.

All I was jokingly suggesting is that you put the invoking parens inside the wrapping parens - to avoid https://www.youtube.com/watch?v=taaEzHI9xyY#t=34m33s